webhookbot/webhookbot/handler.go

```diff
@@ -93,9 +94,14 @@ func (h *Handler) handleCreate(cmd string, msg chat1.MsgSummary) error {
 	convID := msg.ConvID
 	toks := strings.Split(cmd, " ")
 	if len(toks) != 3 {
-		return fmt.Errorf("invalid number of arguments, must specify a name")
+		h.ChatEcho(convID, "invalid number of arguments, must specify a name")
+		return nil
 	}
 	if isAdmin, err := h.checkAdmin(msg); err != nil || !isAdmin {
+		if !isAdmin {
+			h.ChatEcho(convID, "only team admins can create web hooks")
+			return nil
```

Comment:
If `checkAdmin` returns an error, wouldn't the default value of `isAdmin` be false? If so, we would never return that error.